tests/configs/tls13-only.h

```diff
@@ -21,17 +21,10 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-#if defined(MBEDTLS_SSL_ENCRYPT_THEN_MAC)
-#undef MBEDTLS_SSL_ENCRYPT_THEN_MAC
-#endif
 
-#if defined(MBEDTLS_SSL_EXTENDED_MASTER_SECRET)
+#undef MBEDTLS_SSL_ENCRYPT_THEN_MAC
 #undef MBEDTLS_SSL_EXTENDED_MASTER_SECRET
-#endif
-
-#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
 #undef MBEDTLS_SSL_PROTO_TLS1_2
-#endif
 
 #if !defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
You prefer to keep that pre-compiler directive ? We do not really need it either I think (even if this is more beautiful ;-)).

Reply:
Both are okay for me.  
Just my habit, wrap it with `#ifdef `